docs/Installing.asciidoc

```diff
@@ -311,6 +310,23 @@ NFS clients are where openQA workers are running. Run following command:
 mount -t nfs openQA-webUI-host:/var/lib/openqa/share /var/lib/openqa/share
 --------------------------------------------------------------------------------
 
+Needle Caching  
+~~~~~~~~~~~~~~~
+
+If you your network is slow or you experience long time to load needles you
```

Comment:
That the double you even if it makes the sentence sound a bit odd, is trying to give a hint to the user on what to do, I can't think of a better way to offer the solution than the second sentence i wrote.